keras/applications/__init__.py

```diff
@@ -3,3 +3,7 @@
 from .resnet50 import ResNet50
 from .inception_v3 import InceptionV3
 from .xception import Xception
+from .mobilenet import MobileNet
+from .mobilenet import DepthwiseConv2D
+from .mobilenet import DepthwiseConvolution2D
+from .mobilenet import relu6
```

Comment:
These 3 should not be exposed as part of the `applications` module (only as part of the `mobilenet` submodule)